src/core/env-conf.ts

```diff
@@ -1,16 +1,20 @@
+import { config, DotenvConfigOutput } from 'dotenv';
 import { env } from 'node:process';
-import Container, { Token } from 'typedi';
+import { Container, Token } from 'typedi';
 
 export const HOST = new Token<string>('HOST');
 export const PORT = new Token<number>('PORT');
 export const DATABASE_URL = new Token<string>('DATABASE_URL');
 export const CRYPTO_KEY_LENGTH = new Token<number>('CRYPTO_KEY_LENGTH');
 export const CRYPTO_SECRET = new Token<string>('CRYPTO_SECRET');
 
-export namespace EnvConf {
-  export function config(test?: boolean) {
-    require('dotenv').config({ path: test ? '.env.test' : '.env' });
-
+export class EnvConf {
+  static cfg(test?: boolean) {
```

Comment:
que tal? Pode fica mais semanatico de falar hehe

```suggestion
export class EnvConfig {
  static configure(test?: boolean) {
```

Reply:
Eu pensei nisso, mas para digitar e por ser um método achei melhor a escrita assim com apenas três letras que inclusive são próximas no teclado.

Reply:
E tem mais: o **EnvConf** transmite 'Environment Configuration' e então o método é a configuração em si.
Ou não?